package.json

```diff
@@ -19,24 +19,21 @@
     "compile": "rollup -c rollup.config.js"
   },
   "devDependencies": {
+    "@rollup/plugin-commonjs": "^11.1.0",
+    "@rollup/plugin-node-resolve": "^7.1.3",
     "aws-sdk": "^2.3.17",
-    "babel-preset-es2015-rollup": "^1.1.1",
     "eslint": "^6.7.1",
     "exec-chainable": "0.0.3",
-    "nodemon": "^2.0.1",
     "node-riffraff-artefact": "^2.0.1",
-    "rollup": "^0.27.0",
-    "rollup-plugin-babel": "^2.5.0",
-    "rollup-plugin-commonjs": "^2.2.1",
-    "rollup-plugin-node-resolve": "^1.5.0",
+    "nodemon": "^2.0.1",
+    "rollup": "^2.6.1",
```

Comment:
Updated roll-up as `@guardian/pan-domain-node` currently has a circular import dependency which older versions couldn't handle:

We should fix the library but this lets us upgrade independently. The newer version emits a nice warning describing the error:

```
(!) Circular dependencies

node_modules/@guardian/pan-domain-node/dist/src/api.js -> node_modules/@guardian/pan-domain-node/dist/src/panda.js -> node_modules/@guardian/pan-domain-node/dist/src/api.js

node_modules/@guardian/pan-domain-node/dist/src/api.js -> node_modules/@guardian/pan-domain-node/dist/src/panda.js -> /home/travis/build/guardian/pan-domain-lambda/node_modules/@guardian/pan-domain-node/dist/src/api.js?commonjs-proxy -> node_modules/@guardian/pan-domain-node/dist/src/api.js
```